spikeinterface/qualitymetrics/quality_metric_list.py

```diff
@@ -32,7 +32,7 @@
     "firing_rate" : compute_firing_rates,
     "presence_ratio" : compute_presence_ratios,
     "snr" : compute_snrs,
-    "isi_violation" : compute_isi_violations,
+    "isi_violations" : compute_isi_violations,
```

Comment:
```suggestion
    "isi_violation" : compute_isi_violations,
```

Can you change instead this line: https://github.com/SpikeInterface/spikeinterface/blob/master/spikeinterface/widgets/tests/test_widgets.py#L51 ?

Names are more consistent now, i.e., QM names are always singular (while functions are plural)

Reply:
Done.